src/components/Popup/Popup.jsx

```diff
@@ -0,0 +1,30 @@
+import "./Popup.css";
+import React from 'react';
+import { FaCheckCircle } from "react-icons/fa";
+import { FaTimesCircle } from "react-icons/fa";
+import { AiFillExclamationCircle } from "react-icons/ai";
+
+export const Popup = React.forwardRef(({popupType, dialogueText, children}, ref) => {
+  const renderNotificationIcon = () => {
+    switch (popupType) {
```

Comment:
Returns a check icon, x icon or ! icon depending on the type of the popup (success, error and warning respectively)